packages/vuetify/src/components/VDataTable/VDataTable.ts

```diff
@@ -316,7 +316,7 @@ export default VDataIterator.extend({
           on: {
             click: () => this.$set(this.openCache, group, !this.openCache[group]),
           },
-        }, [this.$createElement(VIcon, [isOpen ? 'remove' : 'add'])])
+        }, [this.$createElement(VIcon, [isOpen ? '$vuetify.icons.minus' : '$vuetify.icons.plus'])])
```

Comment:
I feel like icon names should be based on what they are doing, not how do they look like (by default)

Reply:
I agree, kind of. I feel that plus and minus are so generic, and might very well be used for other components in vuetify, that coupling their names too strongly to their usage in this might not be preferable.